clients/admin-ui/src/features/privacy-experience/PrivacyExperiencesTable.tsx

```diff
@@ -150,7 +150,6 @@ export const PrivacyExperiencesTable = () => {
           ),
           header: (props) => <DefaultHeaderCell value="Locations" {...props} />,
           meta: {
-            displayText: "Locations",
```

Comment:
a little "Cargo Cult Coding" happening here, these `displayText` entries were never used on any table except for the Datamap report